daemon/inertia/daemon.go

```diff
@@ -87,7 +87,7 @@ func run(host, port, version string) {
 	mux.HandleFunc("/", gitHubWebHookHandler)
 
 	// Inertia web
-	mux.Handle("/admin/", http.FileServer(http.Dir("./inertia-web")))
+	mux.Handle("/admin/", http.StripPrefix("/admin/", http.FileServer(http.Dir("/app/inertia-web"))))
```

Comment:
Hooray! You can probably switch back to the relative path if you prefer

Reply:
I think absolute path is probably a better idea, just in case ⚒ thanks though!